## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 457348`
Commit: `7fa488570012870ecba2a6b0467c631081bf658f`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tic-80</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tic-80</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>tic-80</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>tic-80</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>tic-80</summary>
<pre>[ 25%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yue_parser.cpp.o
[ 26%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_debug.c.o
[ 26%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_primitive.c.o
[ 27%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_utils.c.o
[ 28%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_value.c.o
[ 28%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_vm.c.o
[ 29%] Building C object CMakeFiles/wren.dir/src/api/wren.c.o
[ 29%] Building C object CMakeFiles/wren.dir/src/api/parse_note.c.o
[ 30%] Linking C static library lib/libwren.a
[ 30%] Built target wren
[ 30%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yue_compiler.cpp.o
CMakeFiles/mruby_vendor.dir/build.make:87: *** missing separator.  Stop.
make[1]: *** [CMakeFiles/Makefile2:533: CMakeFiles/mruby_vendor.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 31%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yuescript.cpp.o
[ 31%] Building CXX object CMakeFiles/yuescript.dir/src/api/yue.cpp.o
[ 32%] Building C object CMakeFiles/yuescript.dir/src/api/parse_note.c.o
[ 33%] Linking CXX static library lib/libyuescript.a
[ 33%] Built target yuescript
make: *** [Makefile:156: all] Error 2</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>tic-80</summary>
<pre>[ 25%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_debug.c.o
[ 25%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_primitive.c.o
[ 26%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_utils.c.o
[ 27%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_value.c.o
[ 28%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yue_parser.cpp.o
[ 28%] Building C object CMakeFiles/wren.dir/vendor/wren/src/vm/wren_vm.c.o
[ 29%] Building C object CMakeFiles/wren.dir/src/api/wren.c.o
[ 29%] Building C object CMakeFiles/wren.dir/src/api/parse_note.c.o
[ 30%] Linking C static library lib/libwren.a
[ 30%] Built target wren
CMakeFiles/mruby_vendor.dir/build.make:87: *** missing separator.  Stop.
make[1]: *** [CMakeFiles/Makefile2:533: CMakeFiles/mruby_vendor.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 30%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yue_compiler.cpp.o
[ 31%] Building CXX object CMakeFiles/yuescript.dir/vendor/yuescript/src/yuescript/yuescript.cpp.o
[ 31%] Building CXX object CMakeFiles/yuescript.dir/src/api/yue.cpp.o
[ 32%] Building C object CMakeFiles/yuescript.dir/src/api/parse_note.c.o
[ 33%] Linking CXX static library lib/libyuescript.a
[ 33%] Built target yuescript
make: *** [Makefile:156: all] Error 2</pre>
</details>
</details>

--- a/pkgs/by-name/ti/tic-80/package.nix
+++ b/pkgs/by-name/ti/tic-80/package.nix
@@ -21,7 +21,7 @@
 }:
 let
   # git rev-list HEAD --count
-  revision = "3016";
+  revision = "3042";
   year = "2025";
 in
 
@@ -29,18 +29,18 @@ stdenv.mkDerivation {
   pname = "tic-80";
   # use an untagged version until upstream tags a new version. We want
   # 'PREFER_SYSTEM_LIBRARIES', and without it tic-80 won't build
-  version = "1.1-unstable-2025-05-26";
+  version = "1.1-unstable-2025-10-21";
 
   src = fetchFromGitHub {
     owner = "nesbox";
     repo = "TIC-80";
-    rev = "663d43924abf6fd7620de6bf25c009ce5b30ab83";
-    hash = "sha256-UjBnXxYZ5gfk58sI1qek5fkKpJ7LzOVmrxdjVgONcXc=";
+    rev = "a2c875f7275541e7724199ce8e504fb578b819a6";
     # TIC-80 vendors its dependencies as submodules. For the following dependencies,
     # there are no (or no compatible) packages in nixpkgs yet, so we use the vendored
     # ones as a fill-in: wasm, squirrel, pocketpy, argparse, naett,
     # sdlgpu, mruby.
     fetchSubmodules = true;
+    hash = "sha256-S/v1WHrvVndW4qHFWRJc3dcQkjw5vCU1/mN0PW9Wfkc=";
   };
 
   # TIC-80 tries to determine the revision part of the version using its Git history.
@@ -101,7 +101,7 @@ stdenv.mkDerivation {
     SDL2
   ];
 
-  meta = with lib; {
+  meta = {
     description = "Free and open source fantasy computer for making, playing and sharing tiny games";
     longDescription = ''
       TIC-80 is a free and open source fantasy computer for making, playing and
@@ -119,9 +119,9 @@ stdenv.mkDerivation {
       channel sound and etc.
     '';
     homepage = "https://github.com/nesbox/TIC-80";
-    license = licenses.mit;
-    platforms = with platforms; linux ++ darwin;
+    license = lib.licenses.mit;
+    platforms = with lib.platforms; linux ++ darwin;
     mainProgram = "tic80";
-    maintainers = with maintainers; [ blinry ];
+    maintainers = with lib.maintainers; [ blinry ];
   };
 }

